### ISSUE TITLE
Rest API => tagi

### ISSUE THREAD

**Issue by pietrach** [allegro/allegro-api#724]
W starym Web API było tak, że tagi można było dodawać bezpośrednio do ofert wpisując odpowiednie frazy a w nowym Rest API widzę, że najpierw trzeba dodać do swojego konta na Allegro i dopiero później pobierać identyfikatory wybranych tagów i przypisywać je do ofert. Czy jest możliwość bezpośredniego wpisywania fraz jako tagów do ofert ? Czy tak jak opisałem w poprzednim zdaniu ?

**Comment by PawelTaberski** [comment]
Tak, jak opisałeś w pierwszym zdaniu trzeba najpierw stworzyć taga a potem można go przypisać do oferty.
